From 853fc888f38d6ad0e189ba54ef648abe0ff5bb60 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Mon, 28 Aug 2017 15:36:17 +0300
Subject: [PATCH] Disable unnamed tmp files

qmake does not work together with pseudo when unnamed temporary files
(O_TMPFILE) are used.

Upstream-Status: Inappropriate [OE specific]
[YOCTO #11996]
Change-Id: I9be92312c3e9a80438c1dc01344cb1ad721ad466
Signed-off-by: Samuli Piippo <samuli.piippo@qt.io>
---
 src/corelib/io/qtemporaryfile_p.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/corelib/io/qtemporaryfile_p.h b/src/corelib/io/qtemporaryfile_p.h
index 46a0d7a..3b418c9 100644
--- a/src/corelib/io/qtemporaryfile_p.h
+++ b/src/corelib/io/qtemporaryfile_p.h
@@ -58,7 +58,7 @@
 #include "private/qfile_p.h"
 #include "qtemporaryfile.h"
 
-#ifdef Q_OS_LINUX
+#if 0
 #  include <fcntl.h>
 #  ifdef O_TMPFILE
 // some early libc support had the wrong values for O_TMPFILE
